source/process/community-guidelines.rst

```diff
@@ -0,0 +1,141 @@
+============================================================
+Mattermost Community Guidelines
+============================================================
+
+In this document we set out some easy-to-follow community guidelines, including:
+
+- `GitHub Help Wanted Issues in Platform Repository`_ - Guidelines for managing Help Wanted issues in GitHub
+- `Community Forums`_ - Guidelines for responding to `Mattermost Forum posts <forum.mattermost.org>`_
+
+----
+
+GitHub Help Wanted Issues in Platform Repository
+---------------------------------------------------------
+
+Goal
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Increase Mattermost usage through more numerous and larger deployments via vibrant open source community by contributing features that would not otherwise be offered
+
+Principles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Contributors get a worthwhile benefit from their contributions from the feature directly, and/or from social recognition
+- Help Wanted issues should have a low barrier to contribute
+
+Creating help wanted issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before creating a help wanted issue in the `platform repository <https://github.com/mattermost/platform>`_, confirm the issue you're planning to add has a JIRA ticket associated with it
+
+- JIRA can be used to prioritize help wanted issues internally
+- each issue goes through `Triage meeting <https://docs.mattermost.com/process/training.html#triage-meeting>`_ to ensure the ticket is clear and is approved by at least one developer and product manager
+
+When creating a help wanted issue, please ensure it contains the following (`see a sample help wanted issue here <https://github.com/mattermost/platform/issues/4755>`_):
+
+- is well-defined and unambiguous
+ 
+ - create the issue with the mindset that the contributor might have no or limited experience with the Mattermost code base and limited exposure to the Mattermost product
+    
+- has an introductory section, with
+ 
+ - an invitation to join the `Contributors channel <https://pre-release.mattermost.com/core/channels/tickets>`_
+ - an invitation to join the `Developers channel <https://pre-release.mattermost.com/core/channels/developers>`_ for technical questions
+ - a link to the `developer’s guide <https://docs.mattermost.com/guides/developer.html>`_ for `machine setup <https://docs.mattermost.com/developer/developer-setup.html>`_ and `developer workflow <https://docs.mattermost.com/developer/developer-flow.html>`_
+ - a link to the `code contribution guidelines <https://docs.mattermost.com/developer/contribution-guide.html>`_
+    
+- has a well-defined difficulty level, both in the description and with the corresponding label
```

Comment:
Yeah, I'm actually starting to think that a visual would be better to describe the different pieces, followed by a template.